contracts/handler/extensible/MarshalLib.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+library MarshalLib {
+    /**
+     * Encode a method handler into a `bytes32` value
+     * @dev The first byte of the `bytes32` value is set to 0x01 if the method is not static (`view`)
+     * @dev The last 20 bytes of the `bytes32` value are set to the address of the handler contract
+     * @param isStatic Whether the method is static (`view`) or not
+     * @param handler The address of the handler contract implementing the `IFallbackMethod` or `IStaticFallbackMethod` interface
+     */
+    function encode(bool isStatic, address handler) internal pure returns (bytes32 data) {
+        data = bytes32(uint256(uint160(handler)) | (isStatic ? 0 : (1 << 248)));
+    }
+
+    function encodeWithSelector(bool isStatic, bytes4 selector, address handler) internal pure returns (bytes32 data) {
+        data = bytes32(uint256(uint160(handler)) | (isStatic ? 0 : (1 << 248)) | (uint256(uint32(selector)) << 216));
+    }
```

Comment:
This and `decodeWithSelector` methods are missing documentation

Reply:
Added 👍🏾